packages/react-devtools-shared/src/devtools/views/root.css

```diff
@@ -1,280 +1,25 @@
-  --light-color-dimmer: #cfd1d5;
-  --light-color-dimmest: #eff0f1;
-  --light-color-error-background: hsl(0, 100%, 97%);
-  --light-color-error-border: hsl(0, 100%, 92%);
-  --light-color-error-text: #ff0000;
-  --light-color-expand-collapse-toggle: #777d88;
-  --light-color-link: #0000ff;
-  --light-color-modal-background: rgba(255, 255, 255, 0.75);
-  --light-color-bridge-version-npm-background: #eff0f1;
-  --light-color-bridge-version-npm-text: #000000;
-  --light-color-bridge-version-number: #0088fa;
-  --light-color-primitive-hook-badge-background: #e5e5e5;
-  --light-color-primitive-hook-badge-text: #5f6673;
-  --light-color-record-active: #fc3a4b;
-  --light-color-record-hover: #3578e5;
-  --light-color-record-inactive: #0088fa;
-  --light-color-resize-bar: #cccccc;
-  --light-color-scheduling-profiler-native-event: #ccc;
-  --light-color-scheduling-profiler-native-event-hover: #aaa;
-  --light-color-scheduling-profiler-native-event-warning: #ee1638;
-  --light-color-scheduling-profiler-native-event-warning-hover: #da1030;
-  --light-color-scheduling-profiler-native-event-warning-text: #fff;
-  --light-color-scheduling-profiler-priority-background: #f6f6f6;
-  --light-color-scheduling-profiler-priority-border: #eeeeee;
-  --light-color-scheduling-profiler-user-timing: #c9cacd;
-  --light-color-scheduling-profiler-user-timing-hover:#93959a;
-  --light-color-scheduling-profiler-react-idle: #edf6ff;
-  --light-color-scheduling-profiler-react-idle-selected:#EDF6FF;
-  --light-color-scheduling-profiler-react-idle-hover:#EDF6FF;
-  --light-color-scheduling-profiler-react-render: #9fc3f3;
-  --light-color-scheduling-profiler-react-render-selected:#64A9F5;
-  --light-color-scheduling-profiler-react-render-hover:#2683E2;
-  --light-color-scheduling-profiler-react-commit: #ff718e;
-  --light-color-scheduling-profiler-react-commit-selected:#FF5277;
-  --light-color-scheduling-profiler-react-commit-hover:#ed0030;
-  --light-color-scheduling-profiler-react-layout-effects:#c88ff0;
-  --light-color-scheduling-profiler-react-layout-effects-selected:#934FC1;
-  --light-color-scheduling-profiler-react-layout-effects-hover:#601593;
-  --light-color-scheduling-profiler-react-passive-effects:#c88ff0;
-  --light-color-scheduling-profiler-react-passive-effects-selected:#934FC1;
-  --light-color-scheduling-profiler-react-passive-effects-hover:#601593;
-  --light-color-scheduling-profiler-react-schedule: #9fc3f3;
-  --light-color-scheduling-profiler-react-schedule-hover:#2683E2;
-  --light-color-scheduling-profiler-react-schedule-cascading:#ff718e;
-  --light-color-scheduling-profiler-react-schedule-cascading-hover:#ed0030;
-  --light-color-scheduling-profiler-react-suspend: #a6e59f;
-  --light-color-scheduling-profiler-react-suspend-hover:#13bc00;
-  --light-color-scheduling-profiler-text-color: #000000;
-  --light-color-scheduling-profiler-react-work-border:#ffffff;
   --light-color-scroll-thumb: #c2c2c2;
   --light-color-scroll-track: #fafafa;
```

Comment:
I left `color-scroll-thumb` and `color-scroll-track` because they are used here:

https://github.com/facebook/react/blob/9f88b5355ba21e06e0a3c4442c5199de354ed1cf/packages/react-devtools-shared/src/devtools/views/Settings/SettingsContext.js#L604-L612